src/Logging/Logging.Console/src/ConsoleLogger.cs

```diff
@@ -156,6 +224,26 @@ private static string GetLogLevelString(LogLevel logLevel)
             }
         }
 
+        private static string GetSyslogSeverityString(LogLevel logLevel)
+        {
+            switch (logLevel)
```

Comment:
Would be nice to have a comment referencing a source for what these priority levels "mean".